Client/Configuration/Fennec.xcconfig

```diff
@@ -7,6 +7,20 @@ MOZ_BUNDLE_DISPLAY_NAME = Fennec ($(USER))
 // Bundle Identifier
 MOZ_BUNDLE_ID = org.mozilla.ios.Fennec
 
+
 // The suffix of the app group identifier used for sharing data across
 // extensions and keychain
 MOZ_GROUP_SUFFIX = org.mozilla.ios.Firefox
+
+// Flag to indicate if we want to include the debug settings bundle or not
+INCLUDE_SETTINGS_BUNDLE = YES
+
+// Adjust SDK Variables
+ADJUST_ENVIRONMENT =
+ADJUST_APP_TOKEN = 
+
+// Leanplum SDK Variables
+LEANPLUM_ENVIRONMENT = development
```

Comment:
I don't think we have a better way to do this. Ideally we do not put any API tokens in Git. For the production builds we must have them in secret BuddyBuild variables that can be replaced in a `buddybuild_prebuild.sh` script. Let me know if I can help with that.